docs/DeveloperGuide.md

```diff
@@ -1097,6 +1097,88 @@ testers are expected to do more *exploratory* testing.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/`<br>
+      Expected: No change to the event list panel. Error message for the invalid field is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present and any argument is empty or invalid.
+
+   1. Test case: `ecreate n/Food distrubting r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`, followed by
+                 `ecreate n/Food distributing r/10 chef r/10 packer sd/23/10/2023 1900 l/hougang dsc/Packing food for the needy`<br>
+      Expected: No change to the event list panel. Error message for duplicate events is shown in the status message<br>
+      The outcome is the same when all compulsory parameters are present, all arguments are valid, and an event with the same name already exists in the event list.
+   
+### Adding a volunteer to an event
+1. Adding a volunteer which is being shown to an event which is being shown
+
+   1. Prerequisites: The event list panel must contain at least one event and the volunteer list panel must contain at least one volunteer.
+
+   1. Test case: `eaddv eid/1 vid/1`<br>
+      Additional Prerequisites:
+         1. The indexes provided are valid.
+         1. The volunteer is not already assigned to the event.
+         1. The volunteer's assigned events do not clash with the current event.<br>
+      
+      Expected: The volunteer is added to the event. Details of the updated event and its current number of assigned volunteers are shown in the status message.
+
+   1. Test case: `eaddv`<br>
+      Expected: No change to the event list or volunteer list panels. Invalid command format error is shown in the status message.<br>
+      The outcome is the same when any of the parameters are missing from the input command, or when their arguments are empty or invalid.
+
+   1. Test case: `eaddv eid/1 vid/1`, followed by `eaddv eid/1 vid/1`<br>
+      Expected: No change to the event list or volunteer list panels. Error message for duplicate volunteer is shown in the status message<br>
+      The outcome is the same when any volunteer is added to an event they are already assigned to.
+
+   1. Test case: `ecreate n/Food packing r/10 chef r/10 packer sd/23/10/2023 1900 ed/23/10/2023 2200 l/hougang dsc/Packing food for the needy` and
+                 `ecreate n/Clean park r/20 cleaner sd/23/10/2023 2000 ed/23/10/2023 2300 l/serangoon dsc/Clean the local park`, followed by
+                 `eaddv eid/1 vid/1` and `eaddv eid/2 vid/1`<br>
+      Note that the two events created are clashing with each other.<br>
+      Expected: No change to the event list or volunteer list panels. Error message for clashing events is shown in the status message<br>
+      The outcome is the same when any volunteer is added to an event which clashes with at least one of their pre-existing events.
```

Comment:
looks lit but i think need to have a prerequisite that event list is empty? else they won't know what `eid/1` and `eid/2` refer to. or else you can do something like `eid/X` like what i did idk hahah

Reply:
aight